src/clj/ataru/hakija/hakija_application_service.clj

```diff
@@ -364,7 +371,8 @@
       (do
         (audit-log/log {:new       input-application
                         :operation audit-log/operation-failed
-                        :id        (util/extract-email input-application)})
```

Comment:
Tässä pitäis varmaan olla käsillä `:application-key` jotain kautta, koska muokkaus. Sitä vois käyttää `:id`:nä.